packages/panzer/disc-fe/src/Panzer_ModelEvaluator.hpp

```diff
@@ -353,6 +353,13 @@ class ModelEvaluator
   void setupAssemblyInArgs(const Thyra::ModelEvaluatorBase::InArgs<Scalar> & inArgs,
                            panzer::AssemblyEngineInArgs & ae_inargs) const;
 
+
+  /**
+   * \brief return a copy of the model evaluators template manager, this is shallow class so pass by value
+   * @return The AssemblyEngine template manager
+   */
+  panzer::AssemblyEngine_TemplateManager<panzer::Traits> getAssemblyEngineTemplateManager() const;
+
```

Comment:
I think you are missing the definition of the function getAssemblyEngineTemplateManager(). I see just the addition of the declaration in this PR. The unit test is building the assembly_engine instead of pulling from ME.